topostats/validation.py

```diff
@@ -97,23 +117,30 @@ def validate_config(config: dict):
                 "save_cropped_grains": Or(
                     True,
                     False,
-                    error="Invalid value in config for 'grainstats.save_cropped_grains, valid values are 'True' or 'False'",  # pylint: disable=line-too-long
+                    error=(
+                        "Invalid value in config for 'grainstats.save_cropped_grains, valid values "
+                        "are 'True' or 'False'"
+                    ),
                 ),
             },
             "dnatracing": {
                 "run": Or(
-                    True, False, error="Invalid value in config for 'filter.run', valid values are 'True' or 'False'"
+                    True,
+                    False,
+                    error="Invalid value in config for 'filter.run', valid values are 'True' or 'False'",
                 )
             },
             "plotting": {
                 "run": Or(
-                    True, False, error="Invalid value in config for 'plotting.run', valid values are 'True' or 'False'"
+                    True,
+                    False,
+                    error="Invalid value in config for 'plotting.run', valid values are 'True' or 'False'",
                 ),
                 "save_format": str,
                 "image_set": Or(
                     "all",
                     "core",
-                    error="Invalid value in config for 'plotting.image_set', valid values are 'all' or 'core'",
+                    error="Invalid value in config for 'plotting.image_set', valid values " "are 'all' or 'core'",
```

Comment:
Is this addition of closing and opening quotes " intended, if so, why did you add it? Just curious!

Reply:
I did it in other places because the file was failing linting with `flake8` due to long lines, doesn't need it on this line though, good spot, I'll remove it in in a separate PR with some other work.